spi/control-plane-adapter-spi/src/main/java/org/eclipse/tractusx/edc/spi/cp/adapter/types/TransferOpenRequest.java

```diff
@@ -0,0 +1,102 @@
+/*
+ *  Copyright (c) 2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *
+ */
+
+package org.eclipse.tractusx.edc.spi.cp.adapter.types;
+
+import org.eclipse.edc.connector.contract.spi.types.offer.ContractOffer;
+import org.eclipse.edc.spi.types.domain.callback.CallbackAddress;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+public class TransferOpenRequest {
```

Comment:
nit: maybe call it `TransferRequest`, since "opening a transfer" is not in our domain language anywhere

Reply:
I just didn't want to mix that with EDC upstream `TransferRequest`, better name are welcome :)